src/components/actionBar/actionBar/ActionBar.styled.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { IconButton } from '@mui/material';
+
+export const ToggleButton = styled(IconButton)(({ theme }) => ({
```

Comment:
```
    transition: 'transform 0.5s ease',

    '&.toggle': {
        transform: 'rotate(180deg)',
    },

    '&.reverse': {
        transform: 'rotate(0)',
    },
```

Use this snippet to fix the initial rotation

Reply:
Added this snippet with minor modification to angles for clockwise rotation for first toggle: https://github.com/nijin-26/bus-ticket-booking-react/pull/24/commits/3e2d69f65aeb832dd70daaced22b1f27440f3297